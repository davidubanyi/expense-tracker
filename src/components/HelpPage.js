import React from "react";

const HelpPage = () => {
  return <div>this is the help page</div>;
};

export default HelpPage;
