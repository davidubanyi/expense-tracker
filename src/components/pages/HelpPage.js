import React from "react";
import Layout from "../Layout";

const HelpPage = () => {
  return (
    <Layout>
      <div>this is the help page</div>
    </Layout>
  );
};

export default HelpPage;
