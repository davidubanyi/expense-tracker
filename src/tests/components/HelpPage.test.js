import React from "react";
import { shallow } from "enzyme";
import HelpPage from "../../components/HelpPage";

test("should render the help page correctly", () => {
  const wrapper = shallow(<HelpPage />);
  expect(wrapper).toMatchSnapshot();
  //expect(wrapper.find("h1").text()).toBe("Expensify");
});
