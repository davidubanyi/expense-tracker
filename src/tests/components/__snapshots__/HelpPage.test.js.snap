// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the help page correctly 1`] = `
<div>
  this is the help page
</div>
`;
